src/components/Clock/Clock.js

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment } from 'react';
+import "./Clock.css";
+
+const Clock = ({ timerDays, timerHours, timerMinutes, timerSeconds }) => {
+    return <Fragment>
+        {/* <div className='date'>
+            <h3 className='text'>MARK THE DATE</h3>
+            <br />
+            <p className='jagriti'>Jagriti will go live on 7 April 2023.</p>
+        </div> */}
+        <section className="timer">
+            <div className="clock">
+                <div className="days">
```

Comment:
code from line 13 to 17 is being used again for hours  minutes and seconds so it will be better if you make a component for it